.changeset/tidy-lies-confess.md

```diff
@@ -0,0 +1,14 @@
+---
+"@razorpay/blade": minor
+---
+
+feat(ActionList): add Virtualization in ActionList
+
+```jsx
+<ActionList isVirtualized>
```

Comment:
Let's check if it's working with Dynamic ActionListItem height, few items having description etc

Reply:
it doesn't. That's the note that I added below